.kokoro/release.sh

```diff
@@ -1,2 +1,6 @@
+#!/bin/bash
+
 # Make sure `JAVA_HOME` is set.
-echo "JAVA_HOME = $JAVA_HOME"
\ No newline at end of file
+echo "JAVA_HOME = $JAVA_HOME"
```

Comment:
[nit] you probably don't want these spaces in the print out because it will be ambiguous whether or not the space came form `$JAVA_HOME` or your printout. Bash is space-sensitive, so it could be a factor in debugging.

Reply:
Good catch! Updated